### PR TITLE
Ed: Clang tidy on ed

### PR DESCRIPTION
**Pull request by AurelienLP**
Same as https://github.com/CanalTP/navitia/pull/3070 but on Ed.

### CODE REVIEW

File: source/ed/build_helper.cpp [hunk 1]
```diff
@@ -250,14 +254,15 @@ VJ& VJ::operator()(const std::string& sp_name,
...
+        if (!b.data->pt_data->networks.empty()) {
             sp->network = b.data->pt_data->networks.front();
+        }
 
         b.sps[sp_name] = sp;
         b.data->pt_data->stop_points.push_back(sp);
         auto sa_it = b.sas.find(sp_name);
         if (sa_it == b.sas.end()) {
-            navitia::type::StopArea* sa = new navitia::type::StopArea();
+            auto* sa = new navitia::type::StopArea();
```
Comment (nonifier) [inline source/ed/build_helper.cpp:265]: Can I have no raw pointers in the code base for :christmas_tree: :santa: :christmas_tree: ?

File: source/ed/ed_persistor.cpp [hunk 2]
```diff
@@ -1045,21 +1055,21 @@ void EdPersistor::insert_vehicle_journeys(const std::vector<types::VehicleJourne
     }
     this->lotus.finish_bulk_insert();
     for (types::VehicleJourney* vj : vehicle_journeys) {
-        std::string values = "";
+        std::string values;
         if (vj->prev_vj) {
-            values = "previous_vehicle_journey_id = " + boost::lexical_cast<std::string>(vj->prev_vj->idx);
+            values = "previous_vehicle_journey_id = " + std::to_string(vj->prev_vj->idx);
```
Comment (nonifier) [inline source/ed/ed_persistor.cpp:1060]: Are you really sure about that ? is it actually the same thing ?

**Review by nonifier — APPROVED** [review]

File: source/ed/build_helper.cpp (hunk 1 above)
Comment (AurelienLP) [reply source/ed/build_helper.cpp:265]: I have this dream too but it won't be as easy as that... Plus I am not sure if the effort is worth the benefit...

File: source/ed/ed_persistor.cpp (hunk 2 above)
Comment (AurelienLP) [reply source/ed/ed_persistor.cpp:1060]: I can check that just in case!

Comment (AurelienLP) [reply source/ed/ed_persistor.cpp:1060]: `vj->prev_vj->idx` is a uint_32_t so it works! :smile:

**Review by xlqian — APPROVED** [review]

**Pull request merged**